magenta/common/testing_lib.py

```diff
@@ -74,7 +75,11 @@ def FromString(string):  # pylint: disable=invalid-name
     return MockStringProto(string)
 
   def SerializeToString(self):  # pylint: disable=invalid-name
-    return 'serialized:' + self.string
+    # protobuf's SerializeToString returns binary string
```

Comment:
Is it simpler/correct to just use bytes(self.string) here and avoid the Python version check?

Reply:
In the Python3, we have to use `bytes` with encoding (like `bytes("Hello", "utf-8")`).  And `bytes` in Python2 can not take encoding arguments.

Python3
```
>>> bytes("Hello world")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: string argument without an encoding
>>> bytes("Hello world", "utf-8")
b'Hello world'
```

Python2
```
>>> bytes("Hello world")
Hello world
>>> bytes("Hello World", "utf-8")
Traceback (most recent call last):                                                                                                                                                                                                                                              
  File "main.py", line 3, in <module>                                                                                                                                                                                                                                           
    print bytes("Hello World!\n", "utf-8")                                                                                                                                                                                                                                      
TypeError: str() takes at most 1 argument (2 given)
```

So Sad.

Reply:
My lack of Python3 knowledge is showing :)